### PR TITLE
 Remove dependency to javax.xml.bind.DatatypeConverter that makes artifactory analyser crashing on Java 11

### PR DESCRIPTION
**Pull request by nhenneaux**
Remove dependency to javax.xml.bind.DatatypeConverter that makes artifactory analyser crashing on Java 11

### CODE REVIEW

File: core/src/main/java/org/owasp/dependencycheck/data/artifactory/ArtifactorySearch.java [hunk 1]
```diff
@@ -41,6 +40,7 @@
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
 import java.util.ArrayList;
+import java.util.Base64;
```
Comment (Stephan202) [inline core/src/main/java/org/owasp/dependencycheck/data/artifactory/ArtifactorySearch.java:43]: The project [targets Java 8](https://github.com/jeremylong/DependencyCheck/blob/master/pom.xml#L426) so this should work :+1:.

**Review by Stephan202 — APPROVED** [review]

File: core/src/main/java/org/owasp/dependencycheck/data/artifactory/ArtifactorySearch.java (hunk 1 above)
Comment (nhenneaux) [reply core/src/main/java/org/owasp/dependencycheck/data/artifactory/ArtifactorySearch.java:43]: Indeed but with this change it can also work with 11

Comment (Stephan202) [reply core/src/main/java/org/owasp/dependencycheck/data/artifactory/ArtifactorySearch.java:43]: I know ;). Just pointing out that this PR doesn't fix Java 11 compatibility at the expense of older still-supported releases. :)

Context for why this change is necessary is [JEP 320](https://openjdk.java.net/jeps/320).

**Comment by jeremylong** [comment]
Thanks for the PR!

**Pull request merged**